IBPSA/Fluid/HeatPumps/ModularReversible/RefrigerantCycle/BaseClasses/PartialRefrigerantCycle.mo

```diff
@@ -0,0 +1,145 @@
+          {2,-108},{2,-72},{76,-72},{76,-78}},color={0,0,127}));
+  connect(redQCon.y, QCon_flow) annotation (Line(points={{70,-101},{70,-116},{-80,
+          -116},{-80,-130}},color={0,0,127}));
+  connect(iceFacCal.iceFac, proRedQEva.u1)
+    annotation (Line(points={{-81.2,-50},{-36,-50},{-36,-78}},
+                                                             color={0,0,127}));
+  connect(iceFacCal.sigBus, sigBus) annotation (Line(
+      points={{-98.08,-50},{-110,-50},{-110,120},{1,120}},
+      color={255,204,51},
+      thickness=0.5));
+  annotation (Icon(coordinateSystem(extent={{-120,-120},
+            {120,120}}),                                        graphics={
+                                Rectangle(
+        extent={{-120,-120},{120,120}},
+        lineColor={0,0,255},
+        fillColor={255,255,255},
+        fillPattern=FillPattern.Solid),   Text(
+          extent={{-57.5,-35},{57.5,35}},
+          pattern=LinePattern.Dash,
+          textString="%name", origin={2.5,153},
+          rotation=180)}),Diagram(coordinateSystem(extent={
+            {-120,-120},{120,120}})),
+    Documentation(revisions="<html><ul>
+  <li>
+    <i>October 2, 2022</i> by Fabian Wuellhorst:<br/>
+    Adapted based on IBPSA implementation. Mainly, the iceFac is added directly
+    in this partial model (see issue <a href=
+    \"https://github.com/ibpsa/modelica-ibpsa/issues/1576\">#1576</a>)
+  </li>
+  <li>
+    November 26, 2018 by Fabian Wuellhorst:<br/>
+    First implementation (see issue <a href=
+    \"https://github.com/RWTH-EBC/AixLib/issues/577\">AixLib #577</a>)
+  </li>
+</ul>
+</html>", info="<html>
+<p>
+  Partial model for calculation of electrical power consumption
+  <code>PEle</code>, condenser heat flow <code>QCon_flow</code>
+  and evaporator heat flow <code>QEva_flow</code> based on the
+  values in the <code>sigBus</code> for a refrigerant machine.
+</p>
+<h4>Frosting performance</h4>
+<p>
+  To simulate possible icing of the evaporator on air-source devices, the
+  icing factor <code>iceFac</code> is used to influence the outputs.
+  The factor models the reduction of heat transfer between refrigerant
+  and source. Thus, the factor is implemented as follows:
+</p>
+<p>
+  <code>QEva_flow = iceFac * (QConNoIce_flow - PEle)</code>
+</p>
+<p>
+  With <code>iceFac</code> as a relative value between 0 and 1:</p>
+<p><code>iceFac = kA/kA_noIce</code></p>
+<p>Finally, the energy balance must still hold:</p>
+<p><code>QCon_flow = PEle + QEva_flow</code> </p>
+<p>
+  You can select different options for the modeling of the icing factor or
+  implement your own approach.
+</p>
+
+<h4>Scaling factor</h4>
+<p>
+  Furthermore, different designs of the refrigerant machine
+  are modeled via a scaling factor <code>scaFac</code>.
+  To linearly scale the outputs of the model according to the specified
+  <code>QUse_flow_nominal</code>, children of this partial model must
```

Comment:
The documentation of various models refers to `QUse_flow_nominal` and `QUseNoSca_flow_nominal`. Can you please search all files for these strings and update it to the right parameter names.